container-storage-setup.1

```diff
@@ -0,0 +1,218 @@
+     \f[B]CONTAINER_ROOT_LV_MOUNT_PATH=/var/lib/containers\f[]
+     would carve out a logical volume, format it with an XFS filesystem
+     and mount it on /var/lib/containers.
+
+     Note: DOCKER_ROOT_VOLUME is deprecated. Specifying
+     DOCKER_ROOT_VOLUME and CONTAINER_ROOT_LV_MOUNT_PATH at the same
+     time is not allowed.
+
+CONTAINER_ROOT_LV_SIZE:
+     Specify the desired size for CONTAINER_ROOT_LV_MOUNT_PATH
+     root volume. It defaults to 40% of all free space.
+
+     CONTAINER_ROOT_LV_SIZE can take values acceptable to
+     \f[B]lvcreate -L\f[] as well as some values acceptable to
+     \f[B]lvcreate -l\f[]. If user intends to pass values acceptable
+     to \f[B]lvcreate -l\f[], then only those values which contains "%"
+     in syntax are acceptable.  If value does not contain "%" it
+     is assumed value is suitable for \f[B]lvcreate -L\f[].
+
+     Note: If both STORAGE_DRIVER=devicemapper and
+     CONTAINER_ROOT_LV_MOUNT_PATH is set, container-storage-setup
+     would set up the thin pool for devicemapper first,
+     followed by extra volume. e.g if free space in the
+     volume group is 10G, devicemapper thin pool size
+     would be 4G (40% of 10G) and extra volume would be
+     2.4G (40% of 6G).
+
+     Note: DOCKER_ROOT_VOLUME_SIZE is deprecated. Specifying
+     DOCKER_ROOT_VOLUME_SIZE and CONTAINER_ROOT_LV_SIZE at the same
+     time is not allowed.
+
+
+Options below should be specified as values acceptable to \f[B]lvextend -L\f[].
+
+ROOT_SIZE: The size to which the root filesystem should be grown.
+
+DATA_SIZE: The desired size for container runtime thin pool data LV.
+	Defaults: 40% free space in the VG after the root LV and container
+	runtime metadata LV have been allocated/grown.
+
+	DATA_SIZE can take values acceptable to \f[B]lvcreate -L\f[] as well as
+	some values acceptable to \f[B]lvcreate -l\f[]. If user intends to pass
+	values acceptable to \f[B]lvcreate -l\f[], then only those values which
+	contains "%" in syntax are acceptable.  If value does not contain
+	"%" it is assumed value is suitable for \f[B]lvcreate -L\f[].
+
+MIN_DATA_SIZE: Specifies the minimum size of the thin pool data LV. If
+	sufficient free space is not available, the pool creation will
+	fail.
+
+	Value should be a number followed by a optional suffix.
+	"bBsSkKmMgGtTpPeE" are valid suffixes. If no suffix is specified
+	then value will be considered as megabyte unit.
+
+	Both upper and lower case suffix represent same unit of size.
+	Use suffix B for Bytes, S for sectors as 512 bytes, K for
+	kibibytes (1024 bytes), M for mebibytes (1024 kibibytes), G for
+	gibibytes, T for tebibytes, P for pebibytes and E for exbibytes.
+
+\f[B]Sample\f[]
+
+A simple, sample INPUTFILE:
+
+DEVS=/dev/vdb
+
+DATA_SIZE=8GB
+
+.fi
+
+.SH "SEE ALSO"
+.BR atomic "(1)"
+
+.SH HISTORY
+
+.PP
+November 2014, originally compiled by Joe Brockmeier <jzb@redhat.com>
+based on comments in Andy Grimm's <agrimm@redhat.com> script.
+.SH AUTHORS
```

Comment:
You have done enough changes to this man page that you probably should add your name to it now as well.